tools/bazel.rc

```diff
@@ -38,3 +38,6 @@ build:clang-msan --copt -fsanitize=memory
 build:clang-msan --linkopt -fsanitize=memory
 build:clang-msan --define tcmalloc=disabled
 build:clang-msan --copt -fsanitize-memory-track-origins=2
+
+# Test options
+test --test_env=HEAPCHECK --test_env=PPROF_PATH
```

Comment:
Nothing breaks if `pprof` isn't installed, and the heap checker still runs.  Setting `--test_env=HEAPCHECK=normal --test_env=PPROF_PATH` should be fine in the `bazel.rc`.  Note: If `HEAPCHECK` is explicitly set to `normal` in `bazel.rc` the user cannot override it by just exporting `HEAPCHECK` in their local environment.  The user will have to explicitly set `--test_env=HEAPCHECK` (grabbing the env variable) or `--test_env=HEAPCHECK=[user_setting]` (modifying it in the command directly) to override the `normal` setting.  Whatever we choose, I'll add it into the docs.